client/web/src/components/Header/style.css

```diff
@@ -16,3 +16,7 @@ header {
   position: absolute;
   right: 3vw;
 }
+
+.nextui-c-hhqfap-ieQnWyR-css{
```

Comment:
`hhqfap-ieQnWyR` this looks really odd

Reply:
"nextui-c-hhqfap-ieQnWyR" is the class name for the dropdown button 

but yea ,  i noticed that and i found out that 

" _The class names generated by NextUI are meant to be concise and optimized for performance, and they are typically automatically generated using a hashing algorithm to ensure uniqueness and minimize the CSS output size_."  dont know if thats true or not but will see if i can rename them to something more descriptive